src/app/filecoin-plus/allocators/data/allocatorTable.tsx

```diff
@@ -0,0 +1,59 @@
+import { ArrowUpRight } from '@phosphor-icons/react/dist/ssr'
+import { createColumnHelper } from '@tanstack/react-table'
+
+import { CustomLink } from '@/components/CustomLink'
+import { Icon } from '@/components/Icon'
+
+import type { AllocatorProps } from '../components/AllocatorsList'
+
+const columnHelper = createColumnHelper<AllocatorProps>()
+
+export function mapAllocatorsToTableData() {
```

Comment:
This file is under a folder called `data`; the file itself is called `allocatorTable`; and the function being exported in `mapAllocatorsToTableData` - I find it confusing - What is this actually?

Reply:
I think we can call this `AllocatorsTableColumns` and move it under the local `components` folder.